aws/table_aws_route53_record.go

```diff
@@ -180,22 +180,6 @@ func listRoute53Records(ctx context.Context, d *plugin.QueryData, _ *plugin.Hydr
 		}
 
 		for _, record := range op.ResourceRecordSets {
-			// The StartRecordName and StartRecordType input parameters only tell
```

Comment:
@ParthaI can you please describe a bit more how the start record name/type matching was causing the bug? Can you please also provide some example inputs/results to demonstrate? Thanks!

Reply:
@cbruno10 Yes,  the start record name/type matching was causing the bug. As per the API behavior, we must have to pass both the last record name and type, as in the input param, if we are performing the pagination.

[which is already been handled](https://github.com/turbot/steampipe-plugin-aws/pull/1375/files#diff-014c84a973889489a14dc5bfbdf67b1e2670e42393ee40b6dd917cbc90ca5a40L209) post the logic which we removed. The newly added code was not being executed due to `continue` [statement](https://github.com/turbot/steampipe-plugin-aws/pull/1375/files#diff-014c84a973889489a14dc5bfbdf67b1e2670e42393ee40b6dd917cbc90ca5a40L191)

Reply:
We made the API call, Initially, there was no record name/type in the input param, so the [newly added code](https://github.com/turbot/steampipe-plugin-aws/pull/1375/files#diff-014c84a973889489a14dc5bfbdf67b1e2670e42393ee40b6dd917cbc90ca5a40L209) was executed properly, and setting the record name/type, while we were fetching the second-page result then the [if condition](https://github.com/turbot/steampipe-plugin-aws/pull/1375/files#diff-014c84a973889489a14dc5bfbdf67b1e2670e42393ee40b6dd917cbc90ca5a40L189) was satisfying and was not letting the below code to be executed, it was resulting the wrong output.

Reply:
I was unable not replicate the `InvalidInput` error as per the query mentioned in the issue description, but if we will set the name and type incorrectly, we will get the `InvalidInput` error for that. So added that to ignore config. The removed code resulted in the wrong `count` output.

Reply:
Should we ignore the `InvalidInput` error though, since the table code controls how the start record type and name input parameters are used. So ideally, our code should handle how those input parameters are set (based on the query quals) in a way that we never hit the `InvalidInput` error. If we're unable to reproduce a specific scenario that replicates the issue, we should not just add general handling that ignores it, as it really masks the issue.

And can you please elaborate **how** the removed code resulted in the wrong `count` output? It may be my misunderstanding, but I'm still unclear as to how that affects count for queries that don't specify any quals.